plugins/org.locationtech.udig.project/src/org/locationtech/udig/project/internal/render/impl/RenderJob.java

```diff
@@ -345,6 +389,20 @@ private boolean isUsingLocalResources(Renderer renderer) {
             }
         }
 
+        private boolean isConflictingGeoResources(Renderer renderer) {
+            try {
+                String conflictResolutionMarker1 = (String) renderer.getContext().getGeoResource()
+                        .getPersistentProperties().get("RENDERING_CONFLICT_RESOLUTION_MARKER"); //$NON-NLS-1$
+                String conflictResolutionMarker2 = (String) getRenderer().getContext()
+                        .getGeoResource().getPersistentProperties()
+                        .get("RENDERING_CONFLICT_RESOLUTION_MARKER"); //$NON-NLS-1$
+                return (conflictResolutionMarker1 != null && conflictResolutionMarker2 != null
+                        && conflictResolutionMarker1.equals(conflictResolutionMarker2));
+            } catch (Exception e) {
+                return false;
+            }
+        }
```

Comment:
.. which checks for 'RENDERING_CONFLICT_RESOLUTION_MARKER' value. Where is the key used to set it in perstistentProperties of GeoResource? I guess I miss something ..